client/client_test.go

```diff
@@ -360,6 +364,113 @@ func (s *ClientSuite) TestNewRoot(c *C) {
 	}
 }
 
+// startTUFRepoServer starts a HTTP server to serve a TUF Repo.
+func startTUFRepoServer(baseDir string, relPath string) (net.Listener, error) {
+	serverDir := filepath.Join(baseDir, relPath)
+	l, err := net.Listen("tcp", "127.0.0.1:0")
+	go http.Serve(l, http.FileServer(http.Dir(serverDir)))
+	return l, err
+}
+
+// newClientWithMeta creates new client and sets the root metadata for it.
+func newClientWithMeta(baseDir string, relPath string, serverAddr string, initWithLocalMetadata bool) (*Client, error) {
```

Comment:
`initWithLocalMetadata` seems to be a left over: it's not used in this function, it's always set to true when called, local meta are always loaded at initialisation